pkg/types/ticket/custom_fields.go

```diff
@@ -79,14 +80,19 @@ func CustomFieldForSchemaField(id string, schema *v2.TicketSchema, value interfa
 		return PickObjectValueField(id, ov), nil
 
 	case *v2.TicketCustomField_PickMultipleObjectValues:
-		vs, ok := value.([]interface{})
-		if !ok {
-			return nil, fmt.Errorf("unexpected value type for custom field: %s %T", id, vs)
+		rawValue, err := json.Marshal(value)
+		if err != nil {
+			return nil, err
 		}
 
+		var vals []interface{}
+		err = json.Unmarshal(rawValue, &vals)
+		if err != nil {
+			return nil, err
+		}
```

Comment:
why this change?

Reply:
While I was writing tests, i realized that i could make it more generic so you could pass json or the proto and have it work. This is really only used for parsing the template.json when using the dev --create-ticket command.